### PR TITLE
wip: add terraform dockerfile, proxmox installatio

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: .github/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,16 @@
+FROM debian:stable-slim
+
+ARG TERRAFORM_VERSION=0.13.5
```
Comment (aamkye) [inline .github/Dockerfile]: This should be arg for further parametrization.

File: .github/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+RUN \
+  apt-get update -y && \
+  apt-get install unzip wget -y
+
+
+RUN wget --progress=dot:mega https://releases.hashicorp.com/terraform/${TERRAFORM_VERSION}/terraform_${TERRAFORM_VERSION}_linux_amd64.zip
```
Comment (aamkye) [inline .github/Dockerfile]: No need for fancy progress in CLI.

File: .github/install_proxmox_ve.sh [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    cd example
+    sed -i 's/10.10.10.2/<NEW_ADRESS>/g' Vagrantfile
+    vagrant up --no-destroy-on-error --provider=virtualbox
+}
+
+packages_installation
+vagrant_installation
+virtualbox_installation
+packer_installation
+proxmox_installation
```
Comment (aamkye) [inline .github/install_proxmox_ve.sh]: No EOL.

File: .github/install_proxmox_ve.sh [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+    sudo apt-get install packer -y
+}
+
+proxmox_installation () {
+    git clone https://github.com/rgl/proxmox-ve
+    cd proxmox-ve
+    make build-virtualbox
+    vagrant box add -f proxmox-ve-amd64 proxmox-ve-amd64-virtualbox.box
+    cd example
+    sed -i 's/10.10.10.2/<NEW_ADRESS>/g' Vagrantfile
```
Comment (aamkye) [inline .github/install_proxmox_ve.sh:28]: What is `<NEW_ADRESS>` ?

File: .github/Dockerfile [hunk 5]
```diff
@@ -0,0 +1,16 @@
+FROM debian:stable-slim
+
+ARG TERRAFORM_VERSION=0.13.5
+
+RUN \
+  apt-get update -y && \
+  apt-get install unzip wget -y
```
Comment (Novirmor) [inline .github/Dockerfile]: Every package in a new line and sort them alphabetically

**Review by Novirmor — CHANGES_REQUESTED** [review]

**Pull request closed without merging**